tests/e2e/testlib/types.go

```diff
@@ -221,31 +221,31 @@ func (c *ChainState) UnmarshalJSON(data []byte) error {
 func UnmarshalProposalWithType(inputMap json.RawMessage, proposalType string) (Proposal, error) {
 	var err error
 	switch proposalType {
-	case "main.TextProposal":
+	case "e2e.TextProposal":
```

Comment:
If you were to run `make test-trace` you would get back:
```
Error reading test file 'tests/e2e/tracehandler_testdata/happyPath.json': e2e.ConsumerAdditionProposal is not a known proposal type
```
but `make e2e-traces` generated files with `e2e.ConsumerAdditionProposal` as their type.

Reply:
@p-offtermatt any idea about this?

Reply:
I don't quite get what you mean - `make test-trace` passes, right? (It passed in the CI on this very PR, and it also passes locally for me)
Is this still relevant? If yes, could you clarify?

Reply:
Maybe related: I fixed a failing unit test in tests/e2e/json_marshal_test.go
371cf97ca529661b2b63da9e2455b4e4a6523aab

Reply:
> I don't quite get what you mean - make test-trace passes, right? (It passed in the CI on this very PR, and it also passes locally for me)
@p-offtermatt It didn't pass with those changes.

> Maybe related: I fixed a failing unit test in tests/e2e/json_marshal_test.go
Thanks for the fix!